tests/integration/search/test_search_api.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import mock
+
+from inspirehep.modules.search.api import LiteratureSearch
+
+
+@mock.patch('inspirehep.modules.search.api.current_app.logger.warning')
+def test_mget_warns_when_no_documents_to_get(w, app):
+    search = LiteratureSearch()
+    search.mget([])
+    w.assert_called_with("TransportError(400, u'action_request_validation_exception', u'Validation Failed: 1: no documents to get;')")
```

Comment:
This assertion will fail if ES library changes its wording. On the other hand, it documents the current expected behaviour. @jacquerie what do you think?